bookchain-miner/src/main.cpp

```diff
@@ -43,5 +46,35 @@ int main(int /*argc*/, const char* /*argv*/[]) {
     std::cout << bookchain::utils::hexifystring(fifthBlock.blockHash()) << std::endl;
     std::cout << bookchain::utils::hexifystring(bloockchain.bloock(testBlockHeight).blockHash()) << std::endl;
 
-    bookchain::http::startMinerHttpServer();
+    std::cout << "Miner thread stopped" << std::endl;
+
+    std::cout << "Looping!" << std::endl;
+    bookchain::PeersList peersList;
+    while (true) {
```

Comment:
For now this is just testing and peering would be in a separate thread from the miner (probably the Node thread) correct?

Reply:
Absolutely! This is literally just a smoke test function right now.